src/LibraryManager/Providers/Unpkg/UnpkgCatalog.cs

```diff
@@ -180,14 +180,14 @@ public async Task<CompletionSet> GetLibraryCompletionSetAsync(string libraryName
                     libraryNameStart.StartsWith("@", StringComparison.Ordinal) &&
                     at < 0 || caretPosition < at) 
                 {
-                    IEnumerable<string> packageNames = await NpmPackageSearch.GetPackageNamesAsync(libraryNameStart, CancellationToken.None);
+                    IEnumerable<Tuple<string, string>> packageNamesAndCurrentVersions = await NpmPackageSearch.GetPackageNamesAndCurrentVersionsAsync(libraryNameStart, CancellationToken.None);
```

Comment:
Instead of changing NpmPackageSearch so drastically, can we get the desired result by using SearchAsync() for package name completion?  That's how the CDNJS catalog does it, and that gives back an ILibraryGroup, so you can use group.Versions.Max() to get the latest version.